styles.css

```diff
@@ -1,10 +1,10 @@
-      body, ul, div, form, input { margin: 0; padding: 0; box-sizing: border-box; }
-      body { font: 13px 'Merriweather Sans', 'Roboto', Helvetica, Arial; overflow: hidden;}
-      form { background: #000; padding: 3px; position: fixed; bottom: 0; width: 100%; }
-      form input { border: 0; padding: 10px; width: 90%; margin-right: .5%; }
-      form button { width: 9%; background: rgb(130, 224, 255); border: none; padding: 10px; }
-      #messages { list-style-type: none; margin: 0; padding: 0; width: 100%;}
-      #messages li { padding: 5px 10px; width: 100%;}
-      #messages li:nth-child(odd) { background: #eee; }
-      #messages li:nth-child(even) { background: #fafafa; }
-      #users { position: fixed; top: 15px; right: 15px; background-color: rgba(0, 0, 0, 0.4); padding: 10px; border-radius: 5px;}
+body, ul, div, form, input { margin: 0; padding: 0; box-sizing: border-box; }
+body { font: 13px 'Roboto', Helvetica, Arial; overflow: auto;}
```

Comment:
@toby56 Can you change `overflow` to `overflow-x` and keep it as hidden? Also, please test what happens on long messages - if we put it all in a container `div` can it wrap?